ansible/roles/openshift_template_deployer/README.md

```diff
@@ -0,0 +1,75 @@
+openshift_volume_provisioner
```

Comment:
This file needs to be updated with the variables that are now being used, and the purpose of it.